### ISSUE TITLE
Camera not available

### ISSUE THREAD

**Issue by r2381** [labexp/osmtracker-android#441]
Although I have not made any changes to the settings (as far as I know), the camera cannot be used while recording a track. The following error message appears as soon changing to the camera.

Foto lässt sich nicht speichern
Kamera ist nicht berechtigt, Fotos an diesem Ort zu speichern

```
Photo cannot be saved
Camera is not authorised to save photos to this location
```

Device: Pixel 4a, Android 13

**Comment by jamescr** [comment]
Indeed that is a known issue of [version 2024.07.09](https://github.com/labexp/osmtracker-android/releases/tag/v2024.07.09).

**Comment by ChildishGiant** [comment]
This is a core feature of the app, it'd be great if the relevant PR #449 could be reviewed as I'm having to use an older version due to this.

**Comment by Binnette** [comment]
Fixed by @Zahnstocher in #449
